test_scripts/API/SetAudioStreamingIndicator/062_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_null_permission.lua

```diff
@@ -0,0 +1,135 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [GeneralResultCode] DISALLOWED. A request comes with appID which has "null" permissions in Policy Table
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- In case PolicyTable has "<appID>": "null" in the Local PolicyTable for the specified application with appID,
+-- PoliciesManager must return DISALLOWED resultCode and success:"false" to any RPC requested by such <appID> app.
+--
+-- 1. Used preconditions
+-- SDL is built with Proprietary or empty flag.
+-- Allow SetAudioStreamingIndicator RPC by policy for HMI levels: "BACKGROUND", "NONE", "LIMITED", "FULL"
+-- Register and activate navi application.
+-- Perform PTU with appID = null permissions
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response("DISALLOWED", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local json = require('json')
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"NONE", "LIMITED", "BACKGROUND", "FULL"}, "SetAudioStreamingIndicator")
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: creates PTU files
+! SetAudioStreamingIndicator_group1.json: assign application with null permissions.
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "files/SetAudioStreamingIndicator_group1.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all")
+  file:close()
+
+  local data = json.decode(json_data)
+  if(data.policy_table.functional_groupings["DataConsent-2"]) then
+    data.policy_table.functional_groupings["DataConsent-2"].rpcs = { json.null }
```

Comment:
@istoimenova Please use

> data.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null

to get correct preloaded in view:

>         "DataConsent-2": {
>             "user_consent_prompt": "DataConsent",
>             "rpcs": null
>         },

In other case you will get:

>   	"DataConsent-2": {
>   		"rpcs": [null],
>   		"user_consent_prompt": "DataConsent"
>              }